Automattic/wc-calypso-bridge
# Deactivate WooCommerce Services if either Woo Shipping or Woo Tax is active

### Changes proposed in this Pull Request:



Closes #1457.

This PR replicates what was done in #1430 to ensure the WooCommerce Shipping & Tax extension is not running if either Woo Shipping or Woo Tax is present but only on Woo Express sites.






### How to test the changes in this Pull Request:




SSH into your dev site and tail -f /tmp/php-errors - Monitor for warnings, fatals related to this PR

1. Set up a Woo Express dev site by following the instructions in the "How to set up an effective development environment for WooExpress & free trial?" section of this page: PdibGW-Uj-p2
2. Make sure you are not sandboxing your WPCOM payment methods.
3. Check out this branch and sync it to your Woo Express dev site.
4. On your test site, install the following plugins:
   - Woo Shipping: https://github.com/woocommerce/woocommerce-shipping/releases/download/0.9.0/woocommerce-shipping.zip
   - Woo Tax: https://github.com/woocommerce/woocommerce-tax/releases/download/0.9.0/woocommerce-tax.zip
   - WooCommerce Shipping & Tax (from the WPORG Plugin Directory)
5. Test different combinations of enabling the plugins.
   - Whenever WCS&T is active and another plugin is activated (either Shipping or Tax), WCS&T should get deactivated. Reasoning: p1713528855792869/1713439513.024539-slack-C04KWSNPSE5
   - Whenever Woo Shipping or Woo Tax are active and you try to activate WCS&T, it should not activate.
   - Verify the changes work on Woo Express sites as well as the Entrepreneur plan on WPCOM, but not other WPCOM plans: https://wordpress.com/start/



### Other information:




### FOR PR REVIEWER ONLY:
